lib/renderer.js

```diff
@@ -719,34 +809,10 @@
                         extract: extract
                     };
                 
-                if (padding && (padding.top || padding.left || padding.right || padding.bottom) && extract) {
-                    //shuffle padding out of the extract coordinates so they don't compete
-                    var clip, delta;
-                    if (padding.right) {
-                        clip = pixmap.width - extract.width - extract.x;
-                        delta = Math.min(clip, padding.right);
-                        extract.width += delta;
-                        padding.right -= delta;
-                    }
-                    if (padding.bottom) {
-                        clip = pixmap.height - extract.height - extract.y;
-                        delta = Math.min(clip, padding.bottom);
-                        extract.height += delta;
-                        padding.bottom -= delta;
-                    }
-                    if (padding.left) {
-                        delta = Math.min(extract.x, padding.left);
-                        extract.x -= delta;
-                        extract.width += delta;
-                        padding.left -= delta;
-                    }
-                    if (padding.top) {
-                        delta = Math.min(extract.y, padding.top);
-                        extract.y -= delta;
-                        extract.height += delta;
-                        padding.top -= delta;
-                    }
-                }
+                var canvasSizeScale = 1;
```

Comment:
This is/was trying to figure out if canvas size should scale with image size. Decision for now was canvas is fixed and image size scales.